src/Commands/Esi/Update/Contracts.php

```diff
@@ -79,11 +79,13 @@ public function handle()
 
                 $token = RefreshToken::find($contract->character_id);
 
-                if ($contract->detail->type == 'auction')
-                    CharacterBids::dispatch($token, $contract->contract_id);
+                if (! is_null($token)) {
+                    if ($contract->detail->type == 'auction')
```

Comment:
I think we can merge both condition here and bellow :)

Reply:
Is this what you where suggesting?